Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -2426,6 +2426,298 @@ def update_cluster_settings_command(client: BotoClient, args: Dict[str, Any]) ->
             )
 
 
+class Lambda:
+    service = AWSServices.LAMBDA
+
+    @staticmethod
+    def get_function_configuration_command(client: BotoClient, args: Dict[str, Any]):
+        """
+        Retrieves the configuration information for a Lambda function.
+
+        Args:
+            client (BotoClient): The boto3 client for Lambda service
+            args (Dict[str, Any]): Command arguments including function name and optional qualifier
+
+        Returns:
+            CommandResults: Results of the operation with function configuration details
+        """
+        # Prepare parameters
+        function_name = args.get("function_name")
+        params = {"FunctionName": function_name}
+        if qualifier := args.get("qualifier"):
+            params["Qualifier"] = qualifier
+
+        # Get function configuration
+        response = client.get_function_configuration(**params)
+
+        # Remove ResponseMetadata for cleaner output
+        if "ResponseMetadata" in response:
+            del response["ResponseMetadata"]
+
+        # Create human readable output
+        human_readable = tableToMarkdown(
+            f"Lambda Function Configuration: {function_name}",
+            response,
+            headers=[
+                "FunctionName",
+                "FunctionArn",
+                "Runtime",
+                "CodeSha256",
+                "State",
+                "Description",
+                "RevisionId",
+                "LastModified",
+            ],
+            headerTransform=pascalToSpace,
+            removeNull=True,
+        )
+
+        return CommandResults(
+            outputs_prefix="AWS.Lambda.FunctionConfig",
+            outputs_key_field="FunctionArn",
+            outputs=response,
+            readable_output=human_readable,
+            raw_response=response,
+        )
+
+    @staticmethod
+    def get_function_url_configuration_command(client: BotoClient, args: Dict[str, Any]):
+        """
+        Returns the configuration for a Lambda function URL
+        """
```

Comment:
Use the full doc string command template, including an explanation, the arguments and the returned value.